server/src/test/java/org/spine3/server/aggregate/AggregatesShould.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.aggregate;
+
+import com.google.common.testing.NullPointerTester;
+import com.google.protobuf.StringValue;
+import org.junit.Before;
+import org.junit.Test;
+import org.spine3.server.BoundedContext;
+
+import java.lang.reflect.Constructor;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.spine3.base.Identifiers.newUuid;
+import static org.spine3.server.aggregate.Aggregates.createAggregatePart;
+import static org.spine3.server.aggregate.Aggregates.createAggregateRoot;
+import static org.spine3.server.aggregate.Aggregates.getAggregatePartConstructor;
+import static org.spine3.test.Tests.hasPrivateParameterlessCtor;
+
+/**
+ * @author Illia Shepilov
+ */
+public class AggregatesShould {
+
+    private BoundedContext boundedContext;
+    private AnAggregateRoot root;
+    private String id;
+
+    @Before
+    public void setUp() {
+        boundedContext = BoundedContext.newBuilder()
+                                       .build();
+        id = newUuid();
+        root = new AnAggregateRoot(boundedContext, id);
+    }
+
+    @Test
+    public void not_accept_nulls_as_parameter_values() throws NoSuchMethodException {
+        final Constructor constructor = root.getClass()
+                                            .getDeclaredConstructor(BoundedContext.class,
+                                                                    String.class);
+        final NullPointerTester tester = new NullPointerTester();
+        tester.setDefault(Constructor.class, constructor);
+        tester.setDefault(BoundedContext.class, boundedContext);
+        tester.setDefault(AggregateRoot.class, root);
+
+        tester.testStaticMethods(Aggregates.class, NullPointerTester.Visibility.PUBLIC);
```

Comment:
There is `testAllPublicStaticMethods()`. The `Aggregates` class has methods with `default` visibility level.